### PR TITLE
Add flowerbeds

### PR DESCRIPTION
**Pull request by RedAuburn**
Fixes https://github.com/organicmaps/organicmaps/issues/4434

i tried more colourful colours, but it looked too vibrant. imo this green looks good in most cases.

before/after
image1

### CODE REVIEW

File: data/styles/clear/include/Basemap.mapcss [hunk 1]
```diff
@@ -98,14 +98,15 @@ area|z12-[natural=grassland],
...
-area|z14-[natural=scrub],
 area|z12-[natural=bare_rock],
 area|z12-[landuse=orchard],
 area|z12-[landuse=vineyard],
 area|z12-[landuse=meadow],
 area|z12-[landuse=recreation_ground],
 area|z12-[landuse=village_green],
 area|z12-[landuse=field],
+area|z14-[natural=scrub],
+area|z16-[landuse=flowerbed],
```
Comment (pastk) [inline data/styles/clear/include/Basemap.mapcss:109]: maybe you can also restore scrubs to z12- if you feel like it?
they were demoted by mistake https://github.com/organicmaps/organicmaps/pull/3786#issuecomment-1467947754

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -14525,6 +14525,47 @@
...
+    ko = "화단"
+    lt = Geliu lova
+    mr = फ्लॉवरबेड
+    nb = Blomsterbed
+    nl = Bloemenbed
+    pl = Kwietnik
+    pt = Canteiro de flores
+    pt-BR = Canteiro de flores
+    ro = Pat de flori
+    ru = цветочная клумба
```
Comment (pastk) [inline data/strings/types_strings.txt:14558]: Клумба

**Review by pastk — APPROVED** [review]

**Comment by pastk** [comment]
> i tried more colourful colours, but it looked too vibrant. imo this green looks good in most cases.
> 

Looks OK though we have too many shades of green already :)

Have you tried brighter colors but desaturated? Maybe a hue similar to the cycleway but faded?

File: data/styles/clear/include/Basemap.mapcss (hunk 1 above)
Comment (RedAuburn) [reply data/styles/clear/include/Basemap.mapcss:109]: will do :+1:

**Comment by RedAuburn** [comment]
> > i tried more colourful colours, but it looked too vibrant. imo this green looks good in most cases.
> 
> Looks OK though we have too many shades of green already :)
> 
> Have you tried brighter colors but desaturated? Maybe a hue similar to the cycleway but faded?

how about this:
image1

**Comment by biodranik** [comment]
Do we need them at all? When do users benefit from seeing them? When will they be distracted?

**Comment by pastk** [comment]
> 
> how about this: 

On my screen it looks good, not too catching. Maybe even more light/faded if you're concerned with them standing out too much.

**Comment by pastk** [comment]
> Do we need them at all? When do users benefit from seeing them? When will they be distracted?

E.g. its not possible usually to short-cut through the flowerbeds :)
Presence of flowerbeds in the park indicate that its more of a managed type of park opposed to a more natural forest-like park.
Some people like the flowers and would want to look for a spot to rest closer to them.

I see your concern though.
Actually we can ditch the most of micromapped features without losing much in practicality.
Lately the approach was to add/accept any minor feature or a niche/rare POI (adds to the map size and clutter, but is important to some, though not many, users also)...
Maybe we should have a discussion about it and come up with some guideline..

**Comment by dvdmrtnz** [comment]
IMO a shade of green is better. Pink is too contrasting

**Comment by RedAuburn** [comment]
back to the original green. i think it's useful context, especially for places like parks, but i'm not super fussed :)

**Review by pastk — APPROVED** [review]

**Comment by pastk** [comment]
btw, thanks for keeping commit messages up-to-date and descriptive!
it helps a lot with preparing release notes!

**Review by vng — APPROVED** [review]

**Comment by pastk** [comment]
@RedAuburn rebase please!

**Comment by RedAuburn** [comment]
> @RedAuburn rebase please!

done :)

**Comment by pastk** [comment]
@vng merge?

File: iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings [hunk 3]
```diff
@@ -1590,7 +1590,7 @@
 
 "type.barrier.chain" = "Cadena";
 
-"type.barrier.city_wall" = "Muro de la ciudad";
+"type.barrier.city_wall" = "City Wall";
```
Comment (biodranik) [inline iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:1593]: @RedAuburn can you please debug why your script always overwrites these strings?

Comment (RedAuburn) [reply iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:1593]: argh sorry, it's strange, sometimes it does, sometimes it doesnt. will double check in future

**Pull request merged**